src/js/components/charts/MainHistogram/useMainHistogram.js

```diff
@@ -48,12 +48,12 @@ export default function(width: number, height: number): HistogramChart {
 
     function onFocus(dates: DateTuple) {
       dispatch(search.setSpanFocus(time.convertToSpan(dates)))
-      dispatch(submitSearch({history: false, investigation: false}))
+      dispatch(submitSearch({history: true, investigation: false}))
```

Comment:
We do want to save the search to the "Back and Forward" history when you zoom in or focus on the histogram..